plugin/figure/src/main/java/io/deephaven/figure/FigureWidgetTranslator.java

```diff
@@ -518,6 +520,119 @@ private FigureDescriptor.ChartDescriptor translate(ChartImpl chart) {
+                                        clientAxes.add(makePartitionedTableSourceDescriptor(
+                                                plotHandle, multiXYErrorBarSeries.getYLow(), SourceType.Y_LOW, yAxis));
+                                        clientAxes.add(makePartitionedTableSourceDescriptor(
+                                                plotHandle, multiXYErrorBarSeries.getYHigh(), SourceType.Y_HIGH,
+                                                yAxis));
+                                    }
+
+                                    clientSeries.setLineColor(stringMapWithDefault(mergeColors(
+                                            multiXYErrorBarSeries.lineColorSeriesNameTointMap(),
+                                            multiXYErrorBarSeries.lineColorSeriesNameToStringMap(),
+                                            multiXYErrorBarSeries.lineColorSeriesNameToPaintMap())));
+                                    clientSeries.setPointColor(stringMapWithDefault(mergeColors(
+                                            multiXYErrorBarSeries.pointColorSeriesNameTointMap(),
+                                            multiXYErrorBarSeries.pointColorSeriesNameToStringMap(),
+                                            multiXYErrorBarSeries.pointColorSeriesNameToPaintMap())));
+                                    clientSeries.setLinesVisible(
+                                            boolMapWithDefault(
+                                                    multiXYErrorBarSeries.linesVisibleSeriesNameToBooleanMap()));
+                                    clientSeries.setPointsVisible(
+                                            boolMapWithDefault(
+                                                    multiXYErrorBarSeries.pointsVisibleSeriesNameToBooleanMap()));
+                                    clientSeries.setGradientVisible(
+                                            boolMapWithDefault(
+                                                    multiXYErrorBarSeries.gradientVisibleSeriesNameTobooleanMap()));
+                                    clientSeries.setPointLabelFormat(stringMapWithDefault(
+                                            multiXYErrorBarSeries.pointLabelFormatSeriesNameToStringMap()));
+                                    clientSeries.setXToolTipPattern(
+                                            stringMapWithDefault(
+                                                    multiXYErrorBarSeries.xToolTipPatternSeriesNameToStringMap()));
+                                    clientSeries.setYToolTipPattern(
+                                            stringMapWithDefault(
+                                                    multiXYErrorBarSeries.yToolTipPatternSeriesNameToStringMap()));
+                                    clientSeries.setPointLabel(stringMapWithDefault(
+                                            multiXYErrorBarSeries.pointColorSeriesNameToStringMap(),
```

Comment:
looks like pointColor is being read and used to set pointLabel...
```suggestion
                                            multiXYErrorBarSeries.pointLabelSeriesNameToObjectMap(),
```